tests/unit/AgmExtensionEvaluation.spec.ts

```diff
@@ -11,7 +11,7 @@ const vuetify = new Vuetify({})
 setActivePinia(createPinia())
 const rootStore = useRootStore()
 
-describe('ExtensionRequest', () => {
+describe('ExtensionEvaluation', () => {
   it('displays normally', () => {
     // init store
     rootStore.keycloakRoles = ['']
```

Comment:
Your component doesn't use the store, so you can remove this (and the ones below) and all references to the store above.

Reply:
Removed, thanks.